std_msgs/Header LineDetectionMsg

float32 error_yaw
float32 error_pitch
